### DISCUSSION TITLE
Overrides cannot be ordered

### DISCUSSION THREAD

**Discussion by nicolaerosia** [grafana/grafana#45978] [Dashboards, Panels & Visualizations]
It would be nice to be able to order the overrides after they are created.
When you have 20 overrides, and you want to add one somewhere in between, it can be a pain to remove them and re-add them again.

**Comment by dsotirakis** [comment]
Hello @nicolaerosia, thanks for raising the issue.
Can you please be more specific to what overrides you refer to? Or explain the actual use case you are facing?
Thanks!

**Comment by nicolaerosia** [comment]
sure, an example would be the Timeseries panel, you create a couple of Overrides, and then you would like to reorder them. Why? Because the order of processing affects the end result/render (z-axis).
It would be nice to be able to drag and drop Override 2, move it on top so it becomes Override 1
![image](https://user-images.githubusercontent.com/4127983/151177097-8be652a4-8bd6-4627-a798-39e9b0243f67.png)

@dsotirakis does it make sense?

**Comment by ashharrison90** [comment]
hi @nicolaerosia 👋 

thanks for raising this. agreed, this would be useful. we're tracking feature requests using GitHub discussions so i'll move this to a discussion until we're ready to commit to implementing it.

thanks! 😄

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
